tests/Feature/CategoriesTest.php

```diff
@@ -15,34 +15,39 @@ public function it_returns_correct_data()
     {
         $category = Category::factory()->create();
 
-        $this->graphQL(/** @lang GraphQL */ '
+        $this->graphQL(
+            /** @lang GraphQL */
+            '
             {
                 categories(page: 1) {
                     data {
                         id
                         name
                         type
+                        color
                     }
                     paginatorInfo {
                         hasMorePages
                     }
                 }
             }
-            ')->assertJson([
-                'data' => [
-                    'categories' => [
-                        "data" => [
-                            [
-                                'id' => $category->id,
-                                'name' => $category->name,
-                                'type' => $category->type,
-                            ],
+            '
+        )->assertJson([
+            'data' => [
+                'categories' => [
+                    "data" => [
+                        [
+                            'id' => $category->id,
+                            'name' => $category->name,
+                            'type' => $category->type,
+                            'color' => 'badge-gray'
```

Comment:
same as mentioned above, let's just store the color itself without the `badge-` prefix